src/System.Management.Automation/engine/remoting/fanin/OutOfProcTransportManager.cs

```diff
@@ -1098,7 +1098,6 @@ internal override void CreateAsync()
                 {
```

Comment:
@daxian-dbw, your last commit had 3 failures in `PowerShell-CI-windows`
Basic Job Tests.Background pwsh process should terminate after job is done.Can clean up background pwsh process after job is done
```powershell
Expected $true, but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Job\Jobs.Tests.ps1: line 350
350:             } -IntervalInMilliseconds 300 | Should -BeTrue
```
Basic Job Tests.Background pwsh process should terminate after job is done.Can clean up background pwsh process when job is stopped
```powershell
Expected $true, but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Job\Jobs.Tests.ps1: line 370
370:             } -IntervalInMilliseconds 300 | Should -BeTrue
```
Basic Job Tests.Background pwsh process should terminate after job is done.Can clean up background pwsh process when job is removed
```powershell
Expected $true, but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Job\Jobs.Tests.ps1: line 390
390:             } -IntervalInMilliseconds 300 | Should -BeTrue
```

Reply:
Not sure why it takes that long for the process to be gone on Windows CI ... Will look into the failures.
But good thing is that all other job tests are passing without the scary out-of-memory error :)

Reply:
It turns out on Windows, the process object can still linger around after the process exists, so `Get-Process -Id $processId` may return a dead process. I will update the tests.